docs/NewProject.md

```diff
@@ -2,43 +2,60 @@
      Licensed under the MIT License. -->
      
 ## Adding a new project
-Do the following when adding a new project:
 
-### For all projects
-1. Add a project to the Axe.Windows Solution (`src\axe.windows.sln`).
-2. Right-click on the project and select Properties.
+### All project types
+
+1. Add your new project to the Axe.Windows Solution (`src\AxeWindows.sln`).
+2. Before creating a pull request, verify that Visual Studio can successfully load and build the entire solution in both Debug and Release.
+3. If your project requires NuGet dependencies which need to be installed along side it, update the <dependencies> section of `./src/ci/axe.windows.nuspec`.
+
+### For all .Net Framework projects
+
+1. Right-click on the project and select Properties.
 2. In the Application tab, configure "Target Framework" to use the same .NET Framework version used by the `axe.windows.core` project.
-   - Currently .NET Framework 4.7.1 is the target. 
+   - Currently .NET Framework 4.7.1
 3. In the build tab, set the following for both Debug and Release configurations:
    1. "Warning level" to 4.
    2. "Treat warnings as errors" to "All".
 
-### For *non-test* projects only
-1. Add the following NuGet packages in Visual Studio to enable signing and code analysis:
+#### For *production (not test)* .Net Framework projects
+
+1. Add the following NuGet packages in Visual Studio to enable signing and code analysis:<br>
    ```
    Microsoft.VisualStudioEng.MicroBuild.Core
    Microsoft.CodeAnalysis.FxCopAnalyzers
    ```
 2. Close the solution and use your text editor to make the following changes to your `.csproj` file to properly configure the version and signing options:
-   1. Add the following line with the other `.cs` files):<br>
+   1. Add the following line along side the existing `<Compile>` statements:<br>
    `<Compile Include="$(TEMP)\AxeWindowsVersionInfo.cs" />`
    2. Add the following below the last ItemGroup:<br>
    ```
    <ItemGroup>
-      <DropSignedFile Include="$(OutDir)\[your project name].dll" />
+      <DropSignedFile Include="$(TargetPath)" />
```

Comment:
>DropSignedFile [](start = 7, length = 14)

Do we still need DropSignedFile? I thought it was for digital signing only.

Reply:
The only way I know to find out is to remove it, create a PR, and create a signed build. I don't object, but I think it's out of scope for this PR.